apps/fm/index.html

```diff
@@ -25,13 +25,13 @@
     <div id="container" dir="ltr">
       <div id="frequency-bar">
         <div id="frequency-display">
-          <a id="speaker-switch" href="#speaker" data-speaker-on="false"></a>
-          <a id="bookmark-button" href="#bookmark" data-bookmarked="false"></a>
+          <a id="speaker-switch" href="#speaker" data-speaker-on="false" data-l10n-id="speaker-switch" role="button"></a>
+          <a id="bookmark-button" href="#bookmark" data-bookmarked="false" data-l10n-id="bookmark-button" role="button"></a>
           <div id="frequency">0</div>
         </div>
       </div>
       <div id="dialer-bar">
-        <div id="dialer-container">
+        <div id="dialer-container" data-l10n-id="dialer-container" role="slider" aria-valuemin="87.5" aria-valuemax="108">
```

Comment:
The semantics here is correct but it will require some more work:
* We need to be able to track, set and keep in sync the aria-valuenow attribute.
* The user should be able to adjust the slider with the wheel up and down events based on the step of 0.1. You can see examples of that implemented here:
https://github.com/mozilla-b2g/gaia/blob/master/apps/system/js/task_manager.js#L208
https://github.com/mozilla-b2g/gaia/blob/master/apps/system/js/task_manager.js#L221
https://github.com/mozilla-b2g/gaia/blob/master/apps/system/js/task_manager.js#L481-L502
https://github.com/mozilla-b2g/gaia/blob/master/apps/system/js/task_manager.js#L617-L619